cms/static/sass/views/_import.scss

```diff
@@ -186,6 +186,17 @@
         // TYPE: success
         &.item-progresspoint-success {
 
+          .item-progresspoint-success-date {
+            display: none;
+            margin-left: 5px;
```

Comment:
For anything that specifies left/right directionality, you need to use the `@include` syntax (eg, `@include margin-left($baseline/4);`) so things work well for RTL users. See https://github.com/edx/edx-platform/wiki/RTL-UI-Best-Practices for more detail.